src/main/less/editor.less

```diff
@@ -427,16 +427,110 @@
     top: 0;
     right: 0;
     bottom: 0;
-    width: 25%;
-    border-left: 1px solid #ccc;
     background: #f8f8f8;
-    padding: 1em;
-    min-width: 250px;
+    padding: 32px;
+    width: 360px;
 }
 
 .split {
+    display: table-row;
     .split-child {
-        display: inline-block;
+        display: table-cell;
         width: 50%;
     }
 }
+
+.editor-pipeline-dialog .errors .error {
+    margin: -1em 0 1.2em 0;
+    color: #c00;
+    background: #fff4f4;
+    padding: 16px;
+    border-radius: 3px;
+}
+
+.editor-config-panel > div > * {
+    margin: .2em 0 1em 0;
+}
+
+.editor-config-panel {
+    .stage-name-edit input {
+        background: rgba(255,255,255,0.5);
+        border: 0;
+        border: none;
+        padding: .2em 0;
+        border-bottom: 1px solid #cdd7e3;
+        width: 100%;
+
+        &:focus {
+            outline: none;
+            border-bottom-color: @brand-color;
+        }
+    }
+
+    input[type=text], .Dropdown-button {
+        border-color: rgba(73, 73, 73, 0.25);
+    }
```

Comment:
I'll add a reminder for myself to chat w/ @brody about border colors and/or make appropriate fix in the JDL so you don't have to be overriding anything here.